javascript-algorithms-and-data-structures/learn-debugging-by-building-a-TODO-app/index.html

```diff
@@ -4,103 +4,92 @@
-            name=""
-            id="text-input"
-            value=""
-          />
-
-          <h3 class="modal-heading-text">Due Date</h3>
-          <input
-            type="date"
-            class="form-control"
-            name=""
-            id="date-input"
-            value=""
-          />
-
-          <h3 class="modal-heading-text">Description</h3>
-          <textarea
-            name=""
-            class="form-control"
-            id="textarea"
-            cols="30"
-            rows="5"
-          ></textarea>
-        </div>
-        <div class="modal-footer">
-          <button type="submit" id="add-task-btn" class="btn">Add</button>
-        </div>
-      </form>
-
-      <dialog id="confirm-modal">
-        <form method="dialog">
-          <h2>Are you sure you want exit? All unsaved data will be lost</h2>
-          <div class="modal-btn-container">
-            <button class="btn" value="cancel" formmethod="dialog">
-              Cancel
+              <svg
+                class="close-icon"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 48 48"
+                width="48px"
+                height="48px"
+              >
+                <path
+                  fill="#F44336"
+                  d="M21.5 4.5H26.501V43.5H21.5z"
+                  transform="rotate(45.001 24 24)"
+                />
+                <path
+                  fill="#F44336"
+                  d="M21.5 4.5H26.5V43.501H21.5z"
+                  transform="rotate(135.008 24 24)"
+                />
+              </svg>
+            </button>
+          </div>
+          <div class="add-task-form-body">
+            <label class="add-task-form-label" for="title-input">Title</label>
+            <input
+              required
+              type="text"
+              class="form-control"
+              id="title-input"
+              value=""
+            />
+            <label class="add-task-form-label" for="date-input">Date</label>
+            <input type="date" class="form-control" id="date-input" value="" />
+            <label class="add-task-form-label" for="description-input"
+              >Description</label
+            >
+            <textarea
+              class="form-control"
+              id="description-input"
+              cols="30"
+              rows="5"
+            ></textarea>
+          </div>
+          <div class="add-task-form-footer">
+            <button
+              id="add-task-btn"
+              class="btn large-btn"
+              type="submit"
+              aria-label="add task"
+            >
+              Add Task
             </button>
-            <button class="btn" id="confirm-btn" value="default">Exit</button>
           </div>
         </form>
-      </dialog>
-
-      <div id="tasks-container"></div>
+        <dialog id="confirm-close-dialog">
+          <form method="dialog">
+            <p class="discard-message-text">Discard unsaved changes?</p>
```

Comment:
I think we should update the language because technically a user could close the modal without changing anything first.


<img width="513" alt="Screenshot 2023-08-30 at 9 02 14 AM" src="https://github.com/freeCodeCamp/CurriculumExpansion/assets/67210629/cdf9a398-757b-4496-88bd-6b76ab4b0056">

How about something like this?

<img width="672" alt="Screenshot 2023-08-30 at 9 02 42 AM" src="https://github.com/freeCodeCamp/CurriculumExpansion/assets/67210629/01554b57-c308-4e18-a32e-d6eaaaf711f6">